### PR TITLE
feature: image optimization

### PR DESCRIPTION
**Pull request by emyriounis**
Image Optimization feature

### CODE REVIEW

File: packages/image-redirection/source.zip
Comment (christophebeling) [inline packages/image-redirection/source.zip:1]: Should this be in here?

File: visuals/distribution-diagram.png
Comment (christophebeling) [inline visuals/distribution-diagram.png:1]: Pls redraw with AWS icons: https://www.drawio.com/blog/aws-diagrams

File: visuals/module-diagram.png
Comment (christophebeling) [inline visuals/module-diagram.png:1]: Same here.

File: packages/image-redirection/source.zip
Comment (emyriounis) [reply packages/image-redirection/source.zip:1]: not required anymore, i removed it

File: visuals/distribution-diagram.png
Comment (emyriounis) [reply visuals/distribution-diagram.png:1]: Will be updated by a new PR

File: visuals/module-diagram.png
Comment (emyriounis) [reply visuals/module-diagram.png:1]: Will be updated by a new PR

**Pull request merged**